### PR TITLE
Moved detector start pixel into base and defaulted to 0

### PR DESCRIPTION
**Pull request by jessemapel**
This sets the default to the first image line being the first detector line and the first image sample being the first detector sample.

### CODE REVIEW

**Comment by jessemapel** [comment]
We will also want to re-run the comparisons after this update because it should shift things a bit.

File: tests/pytests/test_lro_drivers.py [hunk 1]
```diff
@@ -46,12 +46,6 @@ def test_spacecraft_name(driver):
 def test_sensor_model_version(driver):
     assert driver.sensor_model_version == 2
 
-def test_detector_start_sample(driver):
-    assert driver.detector_start_sample == 1
-
-def test_detector_start_line(driver):
-    assert driver.detector_start_sample == 1
```
Comment (Kelvinrr) [inline tests/pytests/test_lro_drivers.py]: Why remove the test instead of updating the value?

Comment (jessemapel) [reply tests/pytests/test_lro_drivers.py]: Because this is no longer defined in the concrete driver, so this is just testing the base Driver class.

Comment (jessemapel) [reply tests/pytests/test_lro_drivers.py]: I thought about adding a test for the base Driver class, but I don't think it's worth it

**Pull request merged**